elm-pkg-js/test-ports.js

```diff
@@ -1,11 +1,10 @@
 exports.init =  async function(app) {
 
-     console.log("Hello! Starting_TEST_PORTS")
+    console.log("Hello! Starting_TEST_PORTS")
 
-
-     app.ports.sendData.subscribe(function(data) {
-            console.log("Data from Elm: ", data);
-            //app.ports.receiveData.send("Hey Elm!");
-        });
+    app.ports.sendData.subscribe(function(data) {
+        console.log("Data from Elm: ", data);
+        //app.ports.receiveData.send("Hey Elm!");
+    });
```

Comment:
This wasn't causing a problem but just tidied the alignment to prevent future issues.